test/commands/api/update.test.js

```diff
@@ -16,6 +16,13 @@ describe('invalid api:update command issues', () => {
     })
     .it('runs api:update with no required --file flag')
 
```

Comment:
New test to validate that if we do not pass `--visibility` with a value we get a failure.